config/home.ts

```diff
@@ -0,0 +1,27 @@
+export const homeData = {
```

Comment:
if we're going to stick to `siteConfig` and `cardsConfig` then might as well have this as `homeConfig`.

```suggestion
export const homeConfig = {
```